tests/harness/cfg/cloud/aws/environments.yaml

```diff
@@ -124,11 +145,34 @@ environments:
     deployment: aws-us-east-1-server
     requirements:
       - external-data-validators
-      - aws-us-east-1-sagemaker
+      - external-neptune-experiment-tracker
+      - external-wandb-experiment-tracker
+      - external-slack-alerter
+      - aws-us-east-1-sagemaker-op
       - aws-us-east-1-mlflow
       - aws-us-east-1-seldon
       - aws-us-east-1-kserve
     mandatory_requirements:
       - aws-us-east-1-cloud
       - aws-us-east-1-ecr
       - aws-us-east-1-tekton
+
+  - name: aws-cloud-sagemaker-orchestrator
```

Comment:
Same here, if it is possible to use the sagemaker orchestrator to run tests now in AWS, beautiful !

Reply:
Same as above, it existed at some point